### PR TITLE
Add NetBSD

### PR DESCRIPTION
**Pull request by 0323pin**
Just packaged lobtui for NetBSD and merged the package into pkgsrc main branch.

Commit log for your reference, https://mail-index.netbsd.org/pkgsrc-changes/2024/09/05/msg306698.html

Please consider adding it to the Readme.md

Thanks!

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -26,6 +26,14 @@ You can install [`lobtui`](https://aur.archlinux.org/packages/lobtui) from the A
 paru -S lobtui
 ```
 
+## 🚩 NetBSD
+
+You cab install [`lobtui`](https://https://pkgsrc.se/editors/lobtui) from the official repositories.
```
Comment (pythops) [inline Readme.md]: a small typo here. `can` instead of `cab` :)

Comment (0323pin) [reply Readme.md]: Sorry ...

Comment (0323pin) [reply Readme.md]: Fixed now

**Comment by 0323pin** [comment]
Thank you!

**Pull request merged**